tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/RenderingContent/RenderingContentWithNumeric.spec.ts

```diff
@@ -0,0 +1,41 @@
+﻿import {AliasHelper, test} from '@umbraco/playwright-testhelpers';
+
+const contentName = 'Test Rendering Content';
+const documentTypeName = 'TestDocumentTypeForContent';
+const dataTypeName = 'Numeric';
+const templateName = 'TestTemplateForContent';
+const propertyName = 'Test Numeric';
+let dataTypeData;
+
+test.beforeEach(async ({umbracoApi}) => {
+  dataTypeData = await umbracoApi.dataType.getByName(dataTypeName); 
```

Comment:
Since we only have one test we can move the before each and after each into the test

Reply:
I don't think so. Currently, the code ensured that clean-up steps are executed, even if the test fails. If the precondition steps fail, the tests will not be executed, so time is not wasted.